shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -194,4 +202,17 @@ public static String nameFromContainer(String metric, int containerId) {
     public static String globalMetricName(String stringName) {
         return stringName + "_global";
     }
+
+    public static String failMetricName(String metricName) {
```

Comment:
Please add an explanation in the form of a Javadoc for this method. It's hard to tell what it wants to achieve.

Reply:
Added (was trying to be consist with the rest of the code, and ./gradlew check didn't report missing description)